src/components/search-filters/filter-group.tsx

```diff
@@ -124,7 +124,8 @@ const FilterGroup: React.FunctionComponent<FilterGroupProps> = memo(
                                                     name={`${name} ${type} ${
                                                         isMobile ? 'mobile' : ''
                                                     }`}
-                                                    label={`${name} (${count})`}
+                                                    // label={`${name} (${count})`}
```

Comment:
Is this comment needed?

Reply:
So I think the plan is to have these counts added in a later version of the site, so I wanted to leave this here just as a reminder.